subsys/bluetooth/mesh/proxy.c

```diff
@@ -291,6 +292,12 @@ static void proxy_cfg(struct bt_mesh_proxy_client *client)
 		return;
 	}
 
+	if (bt_mesh_replay_check(&rx, NULL)) {
+		BT_WARN("Replay: src 0x%04x dst 0x%04x seq 0x%06x",
+			rx.ctx.addr, rx.ctx.recv_dst, rx.seq);
+		return;
+	}
+
```

Comment:
This is a bug. Could you split this out into a separate PR, so we can add it to 2.4?